static/js/file_select.js

```diff
@@ -242,4 +244,17 @@ function submitJson(){
     var data = {'template': template, 'json': JSON.stringify(tableData)};
     console.log(data);
     sendTopython(data);
+    removeTableContent();
+}
+
+function removeTableContent() {
+    tableData = null, tableLabels= null;
+    var table = document.getElementById("parsedgrid");
+    table.deleteTHead();
+    table.createTHead();
+    var tableBody = table.getElementsByTagName("tbody")[0];
+
```

Comment:
It may be simpler to just set table.innerHTML value to a blank string here if it has the same effect as these lines